### ISSUE TITLE
OAuthException while signing in!

### ISSUE THREAD

**Issue by n0shake** [mr-karan/Insta-notFollow#1]
This is a great idea. But, I get this error when I try to sign-in:

{"code": 400, "error_type": "OAuthException", "error_message": "Invalid scope field(s): follower_list"}

**Comment by mr-karan** [comment]
Yes. It broke recently,  because  Instagram has decided to change their policy for creating applications. First need to get approved from them for permissions.

**Comment by n0shake** [comment]
Ohh, okay. Off topic, your website is really cool! Can you tell me which language/frameworks/tools did you use?

**Comment by mr-karan** [comment]
Thanks

Used https://github.com/prakhar1989/react-term for a terminal emulator, and blog is powered by Ghost. Using https://github.com/axitkhurana/buster to convert to a static site and deployed on GH Pages

**Comment by lakshay42** [comment]
log in nahi ho raha bhai

**Comment by mr-karan** [comment]
Instagram has changed their OAuth APIs, so my code is deprecated for now. If I get time, I'll update to the new login flow or else feel free to send a PR.
